chaste_codegen/templates/cvode_model.hpp

```diff
@@ -1,52 +1,10 @@
 #ifdef CHASTE_CVODE
-#ifndef {% filter upper %}{{class_name}}_HPP_{% endfilter %}
-#define {% filter upper %}{{class_name}}_HPP_{% endfilter %}
-
-//! @file
-//!
-//! This source file was generated from CellML by chaste_codegen version {{converter_version}}
-//!
-//! Model: {{model_name}}
-//!
-//! Processed by chaste_codegen: https://github.com/ModellingWebLab/chaste-codegen
-//!     (translator: chaste_codegen, model type: CVODE )
-//! on {{generation_date}}
-//!
-//! <autogenerated>
-
-#include "ChasteSerialization.hpp"
-#include <boost/serialization/base_object.hpp>
-#include "AbstractCvodeCell.hpp"
-{% if dynamically_loadable %}#include "AbstractDynamicallyLoadableEntity.hpp"{%- endif %}
-#include "AbstractStimulusFunction.hpp"
-
-class {{class_name}} : public AbstractCvodeCell{%- if dynamically_loadable %}, public AbstractDynamicallyLoadableEntity{%- endif %}
-{
-    friend class boost::serialization::access;
-    template<class Archive>
-    void serialize(Archive & archive, const unsigned int version)
-    {
-        archive & boost::serialization::base_object<AbstractCvodeCell >(*this);
-        {% if dynamically_loadable %}archive & boost::serialization::base_object<AbstractDynamicallyLoadableEntity>(*this);{%- endif %}
-    }
-    
-    // 
-    // Settable parameters and readable variables
-    // 
-    
-public:
-    {%- if default_stimulus_equations["equations"]|length > 0 %}
-    boost::shared_ptr<RegularStimulus> UseCellMLDefaultStimulus();
-    {%- endif %}
-    {%- if use_get_intracellular_calcium_concentration %}
-    double GetIntracellularCalciumConcentration();
-    {%- endif %}
+{% include "Normal/hpp/header_comments" %}
+{% include "Normal/hpp/includes" %}
+{% include "Cvode/hpp/class_def" %}
```

Comment:
If this ends up being the only file in `Cvode/hpp` you might want not to have a separate file, but I don't have a strong opinion on that.